checks/core.py

```diff
@@ -35,6 +35,9 @@ def checkCPU(lines):
         elif ('i3' in cpu[0]):
             return [LEVEL_INFO, "Insufficient Hardware",
                     "Your system is below minimum specs for OBS to run and is too underpowered to livestream using software encoding. Livestreams and recordings may run more smoothly if you are using a hardware encoder like QuickSync, NVENC or AMF (via Settings -> Output)."]
+        elif ('Apple' in cpu[0]):
+            return [LEVEL_INFO, "Apple Silicon Mac",
+                    "You are running OBS Studio on an Apple Silicon Mac."]
```

Comment:
It's just a reminder that when something weird happens, the ARM64 architecture can play a role in it.

Reply:
I commented to @gxalpha about this, which may have prompted this review comment.

The annoyance here is that when using the analyzer via obs-bot, the CPU is listed under a visible "Hardware Check" header. In this usage, the extra Info entry would be superfluous. On the web interface, however, the CPU is not visibly displayed. Since the web interface lacks the prominent display of the CPU and GPU, I suppose we can keep this.